help/actions/_posts/2015-02-09-export.md

```diff
@@ -74,27 +74,35 @@ The same format as above but encoded for use as a rich text document (rtf) which
-**l** | lastname, f. i. | lastname, f. i. | , | & | n author(s), et al. | Vinodkumar, Dharssi, I., Bally, J., Steinle, P., McJannet, D. \\& Walker, J.
-**m** | lastname | lastname | , | | n author(s), et al. | Vinodkumar, Dharssi, Bally, Steinle, McJannet \\& Walker
-**n** | lastname | + | | | | Vinodkumar+
-
-Note: xx is number of authors - 1.
-
-Also note: Formats *L* and *g* differ by a white space. Format *L* has a space after first initial dot, while format *g* does not. 
+There are 16 author formats in the custom format. 
+
+Author formats for **last name** only
+
+| **and**                                                                                                                        	| **format**                                                       	| **specifier** | **abbreviated**       | **example**                                                                                                                    	| **example abbreviated %3.2**                                                            	|
+|--------------------------------------------------------------------------------------------------------------------------------	|------------------------------------------------------------------	|-----------	|----------------------	|--------------------------------------------------------------------------------------------------------------------------------	|-----------------------------------------------------------------------------------------	|
+| text                                                                                                                           	| first author only if<br>number of authors > 2<br>otherwise both authors 	| **%H**     |                      	| -Henneken<br>-Kurtz and Accomazzi                                                                                                  	|  -Henneken Muench<br>-Kurtz and Accomazzi                                                    	|
+| symbol                                                                                                                         	| first author only if<br>number of authors > 2<br>otherwise both authors 	| **%h**     |                      	|  -Henneken<br>-Kurtz & Accomazzi                                                                                                    	|  -Henneken Muench<br>-Kurtz & Accomazzi                                                             	|
+| text                                                                                                                           	| lastname                                                         	| **%M**     | , et al.             	|  -Henneken, Muench, Holm Nielsen, Blanco-Cuaresma, and Accomazzi<br>-Kurtz and Accomazzi                                            	|  -Henneken, Muench et al.-<br>Kurtz and Accomazzi                                            	|
```

Comment:
* Switch `-` and `<br>` the second "example abbreviated" example
* Missing comma in `, et al.` in the "example abbreviated" example?